CMakeLists.txt

```diff
@@ -27,32 +27,33 @@ find_package(geometry_msgs REQUIRED)
 # Include directories
 include_directories(${PROJECT_SOURCE_DIR}/include)
 install(DIRECTORY launch DESTINATION share/${PROJECT_NAME}/)
-target_include_directories(LidarSubscriberLib PRIVATE ${PROJECT_SOURCE_DIR}/include)
+target_include_directories(LMLib PRIVATE ${PROJECT_SOURCE_DIR}/include)
 add_executable(member_function src/member_function.cpp)
 add_executable(create3 src/create3-fastslam.cpp)
 
 # Link dependencies
 
 ament_target_dependencies(member_function rclcpp std_msgs)
-ament_target_dependencies(LidarSubscriberLib rclcpp sensor_msgs std_msgs geometry_msgs visualization_msgs)
+ament_target_dependencies(LMLib rclcpp sensor_msgs std_msgs geometry_msgs visualization_msgs)
 ament_target_dependencies(create3 rclcpp std_msgs irobot_create_msgs nav_msgs sensor_msgs visualization_msgs)
 
 # Optionally define compilation flags
 option(USE_ROB_ODOM "Use robot odometry estimate" ON)
 option(USE_ROB_IMU "Use robot imu for estimate" OFF)
 option(USE_ROB_CTRL "Use robot control for estimate" OFF)
+option(VISUALIZE_ROB_PATH "Visualize Robot Path" ON)
+option (VISUALIZE_LANDMARK_OBSERVATIONS "publishes information to visualize landmark observations in laser frame" OFF)
 
 # Link libraries for create3
 target_link_libraries(member_function FastSLAMLib)
-target_link_libraries(LidarSubscriberLib FastSLAMLib)
-target_link_libraries(create3 FastSLAMLib LidarSubscriberLib)
-target_compile_definitions(create3 PUBLIC USE_ROB_ODOM USE_ROB_IMU USE_ROB_CTRL)
-
+target_link_libraries(LMLib FastSLAMLib)
+target_link_libraries(create3 FastSLAMLib LMLib)
+target_compile_definitions(create3 PUBLIC USE_ROB_ODOM USE_ROB_IMU USE_ROB_CTRL VISUALIZE_ROB_PATH VISUALIZE_LANDMARK_OBSERVATIONS)
 # Install targets
 install(TARGETS
   member_function
   create3
-  LidarSubscriberLib
+  LMLib
   DESTINATION lib/${PROJECT_NAME}
```

Comment:
I don't think this is necessary here, since `LMLib` isn't a ROS executable. I missed it earlier somehow.